assets/app/view/part/location_name.rb

```diff
@@ -154,15 +190,24 @@ def render_part
           ])
       end
 
-      def render_background_box
-        lines = @name_segments.size
-        characters = @name_segments.map(&:size).max
+      def box_dimensions
+        @box_dimensions ||=
+          begin
+            lines = @name_segments.size
+            characters = @name_segments.map(&:size).max
```

Comment:
@name_segments.max(&:size)

Reply:
this gives the segment that is largest, but I want the the size of that largest segment

@name_segments.max(&:size).size ?

Reply:
yea